### PR TITLE
IP address properties should not be truncated to floats

### PR DESCRIPTION
**Pull request by faucct**

### CODE REVIEW

File: spec/unit/cli/validation_spec.rb [hunk 1]
```diff
@@ -60,7 +60,7 @@ class Validator
   end
   describe "#properties_from_string" do
     it "parses properties into a hash" do
-      provided = %w{key1=value key2=1 key3=true key4=FaLsE key5=0777 key6=https://some.website key7=num1and2digit key_8=underscore}
+      provided = %w{key1=value key2=1 key3=true key4=FaLsE key5=0777 key6=https://some.website key7=num1and2digit key_8=underscore key9=127.0.0.1}
```
Comment (tyler-ball) [inline spec/unit/cli/validation_spec.rb]:
```suggestion
      provided = %w{key1=value key2=1 key3=true key4=FaLsE key5=0777 key6=https://some.website key7=num1and2digit key_8=underscore key9=127.0.0.1 key10=1. key11=1.1}
```

File: spec/unit/cli/validation_spec.rb [hunk 2]
```diff
@@ -70,6 +70,7 @@ class Validator
         "key6" => "https://some.website",
         "key7" => "num1and2digit",
         "key_8" => "underscore",
+        "key9" => "127.0.0.1",
```
Comment (tyler-ball) [inline spec/unit/cli/validation_spec.rb]:
```suggestion
        "key9" => "127.0.0.1",
        "key10" => 1.0,
        "key11" => 1.1,
```

**Comment by faucct** [comment]
Done.

**Comment by tyler-ball** [comment]
Thank you @faucct !

**Pull request merged**